web/src/app/components/dashboard/dashboard.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { AuthService } from 'src/app/services/auth.service';
+
+@Component({
+  selector: 'app-dashboard',
+  templateUrl: './dashboard.component.html',
+  styleUrls: ['./dashboard.component.scss']
+})
+export class DashboardComponent implements OnInit {
+
+  constructor(
+    public authService: AuthService,
+    public router: Router,
```

Comment:
I think these can be `private`

Reply:
okay Eddie. Changed it to private

Reply:
@eddiejaoude I have completed working on the review comments. Please review and merge.